it.ts

```diff
@@ -1350,62 +1350,62 @@ export const IT = {
-    ExportSaveForBackup: "Backup Your Save",
+        "Scegli questa opzione se vuoi salvare tra dispositivi. Questo esporterà il tuo diritto allo scambio dal salvataggio- potrai continuare a scambiare dopo aver importato il file. Il tuo dispositivo attuale non potrà scambiare senza aspettare per un periodo di cooldown",
+    ExportSaveForBackup: "Esegui il backup del tuo salvataggio",
     ExportSaveForBackupDesc:
-        "Choose this if you simply want to take a backup of your progress. This will leave your right to trade on your current device - importing the backup save file will not restore the right to trade without waiting for a cooldown period",
-    CheckRightToTrade: "Check Right To Trade",
-    RightToTradeValid: "Right To Trade",
-    RightToTradeInvalid: "No Right To Trade",
+        "Scegli questa opzione se vuoi semplicemente un backup dei tuoi progressi. Questa opzione ti lascierà il diritto di scambio sul tuo dispositivo attuale - importare il file di backup non annullerà il cooldown",
+    CheckRightToTrade: "Controlla il diritto alo scambio",
+    RightToTradeValid: "Diritto allo scambio",
+    RightToTradeInvalid: "Nessun diritto allo scambio",
     RightToTradeInvalidDesc:
-        "This is because your right to trade has been exported to another save file - you should import that save file to restore your right to trade. If you have lost that save file, you can force restore your right to trade on this save but you will be subject to a cooldown and your other save will lose the right to trade",
-    ForceRestoreRightToTrade: "Force Restore (%{cooldown}-hour cooldown)",
-    ImportSaveRestoreFail: "Failed to connect to the server, please check your internet connection",
-    DefenseCommandConvertDesc: "Convert resources into damage at its current trade center price",
-    MousewheelSensitivity: "Mousewheel Zoom Sensitivity",
-    MousewheelSensitivityDesc: "Adjust the sensitivity when scrolling mousewheel to zoom in/out",
-    PlayerTradeFilterWhatIHave: "Filter What I Have",
+        "Questo avviene perchè il tuo diritto allo scambio è stato esportato assieme ad un altro file - dovresti importare quel salvataggio per resettare il tuo diritto allo scambio. Se hai perso quel file puoi forzare il tuo diritto allo scambio in questo salvataggio, ma sarai soggetto a  un cooldown e il tuo altro salvataggio perderà il diritto allo scambio",
+    ForceRestoreRightToTrade: "Reset forzato (%{cooldown}-ore di cooldown)",
+    ImportSaveRestoreFail: "Connessione al server fallita, per favore controlla la tua connessione a internet",
+    DefenseCommandConvertDesc: "Converte risorse in danno uguale al loro prezzo nel mercato",
+    MousewheelSensitivity: "Sensibilità della rotella del mouse",
+    MousewheelSensitivityDesc: "Aggiusta la sensibilità del mouse quando esegui zoom in e zoom out",
+    PlayerTradeFilterWhatIHave: "Filtra ciò che possiedo",
```

Comment:
PathTemporarilyBlocked:
        "Non puoi costruitrire su questa casella perchè c'è un ondata in corso. Prova di nuovo quando l'ondata sarà finita.",